fundamental-concepts/template-authorization/5_controllers.md

```diff
@@ -0,0 +1,46 @@
+The third UNIX access right is the right to execute an executable file, denoted by **x** in the `ls`
+output. While in UNIX the right to access is per file, in DAML the right to execute is more fine
+grained and denoted per choice of a contract. You give a party the right to execute a choice, by making
```

Comment:
We should explain what a choice is here and how it ties to controllers. Maybe also rename this step as "Choices and controllers".

Reply:
done.